### PR TITLE
Update HRMP Documentation

### PR DESCRIPTION
**Pull request by joepetrowski**
Supersedes https://github.com/w3f/polkadot-wiki/pull/5494

### CODE REVIEW

File: docs/build/build-hrmp-channels.md [hunk 1]
```diff
@@ -7,104 +7,72 @@ keywords: [HRMP, parachain, statemint, proposal, asset hub, statemine]
...
+two channels, one in each direction.
 
-To open a channel from one parachain to another that are not system chains on Polkadot, the
-governance of each parachain needs to declare its intent to open a channel on the Relay Chain, and
-the second chain needs to accept and do the same.
+## Opening HRMP Channels
 
-In order to do this, the calls to be dispatched from both chains are:
+Opening a channel between two parachains is a two-phase process, with one chain first initiating a
+channel request and then the second chain accepting it. When neither chain is a system chain, they
```
Comment (profradha) [inline docs/build/build-hrmp-channels.md]:
```suggestion
channel request and then the second chain accepting it. When neither chain is a [system chain](../learn/learn-system-chains.md), they
```

File: docs/build/build-hrmp-channels.md [hunk 2]
```diff
@@ -7,104 +7,72 @@ keywords: [HRMP, parachain, statemint, proposal, asset hub, statemine]
...
-3.  Technical details of the proposal, including proposal parameters and technical details of this
-    call (On Kusama, most proposals were designed as a batchAll calls):
+:::info
 
-    - A `forceTransfer` of 20 DOT from the Treasury to AssetHub sovereign account.
-    - A `forceOpenhrmpchannel` from AssetHub (1,000) to ParaID.
-    - A `forceOpenhrmpChannel` from ParaID to AssetHub (1,000).
+As of Polkadot and Kusama runtimes 1,001,000, channel establishment with system chains will not
+require a deposit. However, for lower spec versions, each chain in the channel will need a free
+balance of at least the required channel deposits plus the existential deposit.
```
Comment (profradha) [inline docs/build/build-hrmp-channels.md:65]:
```suggestion
:::info

As of Polkadot and Kusama runtimes 1,001,000, channel establishment with system chains will not
require a deposit. However, for lower spec versions, each chain in the channel will need a free
balance of at least the required channel deposits plus the existential deposit.

:::

```

**Review by DrW3RK — APPROVED** [review]

File: docs/build/build-hrmp-channels.md [hunk 3]
```diff
@@ -7,104 +7,72 @@ keywords: [HRMP, parachain, statemint, proposal, asset hub, statemine]
...
+   channel establishment by creating a channel request with a given configuration. Note that the max
+   capacity and max message size must be within the `configuration`'s limits.
 
-Each parachain is to use its own governance process to execute this. The call will be executed on
-the Relay Chain.
+2. `hrmp > hrmpAcceptOpenChannel(sender)`: Accept the channel open request from the given sender.
+
+In order to dispatch a call from its sovereign origin, a parachain may use governance to send the
+encoded call in a `Transact` instruction to the Relay Chain, but it may also execute this logic
+autonomously (e.g. on the notification that a channel was requested).
```
Comment (filippoweb3) [inline docs/build/build-hrmp-channels.md:36]:
```suggestion

The xcm-executor pallet can be forked by parachains' teams and modified to include logic to handle [high-level HRMP instructions](https://paritytech.github.io/xcm-docs/journey/channels-and-bridges.html#hrmpnewchannelopenrequest). These instructions are not included in the default version of the pallet, but can be used for opening, accepting, and closing channels without relying on governance.

```

**Review by filippoweb3 — APPROVED** [review]
Added some info about the possibility of not going through gov

File: docs/build/build-hrmp-channels.md (hunk 3 above)
Comment (profradha) [reply docs/build/build-hrmp-channels.md:36]: Let us add this info as a separate PR, after merging this PR's content that has streamlined information.

**Pull request merged**